syntax = "proto3";

option go_package = "github.com/t10471/go-examples/grpc-examples/go/proto";

package helloworld;

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}